app/pages/HomePage/HomePage.tsx

```diff
@@ -27,6 +32,79 @@ export interface HeroData {
   buttons: ButtonType[];
 }
 
+interface HomePageData {
+  heroSection: HeroData;
+  categoriesSection: FeaturedCategoriesData;
+  opportunitySection: OppEventCardData;
+  eventSection: OppEventCardData;
+  /* Fix naming in Sanity schema */
+  twoColumnContentSections: {
+    twoColumnContentSections: TwoColumnContent;
+  }[];
+}
+
+export const HomePage = () => {
+  const [homePageData, setHomePageData] = useState<HomePageData | null>(null);
+
+  useEffect(() => {
+    const fetchHomePageData = async () => {
+      const query = `*[_type == "homePage" && name == "Home Page"][0]{
+        'heroSection': *[_type == "hero" && name == "Home Hero"][0],
+        categoriesSection {...,'featuredCategoriesSection': featuredCategoriesSection[]->},
+        opportunitySection {...,'opportunityCards': opportunityCards[]->},
+        eventSection {...,'eventCards': eventCards[]->},
+        'twoColumnContentSections': *[_type == 'contentPageType' && name == 'Home']{
+      twoColumnContentSections[0]->
+    }
+      }`;
+
+      const result: HomePageData = await client.fetch(query);
+
+      setHomePageData(result);
+    };
+
+    fetchHomePageData();
+  }, []);
+
+  if (!homePageData) {
+    return <div>Loading...</div>;
+  }
+
+  const {
+    categoriesSection,
+    opportunitySection,
+    eventSection,
+    heroSection,
+    twoColumnContentSections,
+  } = homePageData;
+
+  /* Fix naming in Sanity schema */
+  const twoColumnContentData =
+    twoColumnContentSections[0].twoColumnContentSections;
+
+  return (
+    <>
+      <Hero
+        backgroundImage={builder.image(heroSection.backgroundImage).url()}
+        title={heroSection.title}
+        description={heroSection.description}
+        buttons={heroSection.buttons}
+      />
+      <CategorySection sectionData={categoriesSection} />
+      <OppEventCardSection
+        sectionType="opportunity"
+        sectionData={opportunitySection}
+      />
+      <OppEventCardSection sectionType="event" sectionData={eventSection} />
+      <TwoColumnContentSection {...twoColumnContentData} />
+      {/* Newsletter Component */}
+      <Footer />
+    </>
+  );
+};
+
+// Remove when new categories is created
+// other components are dependent on this list
```

Comment:
Suggestion: add a `TODO` or similar label so this isn't mistaken with explaining behavior. IME these comments sometimes accidentally stick around and become esoteric erratum to be deciphered by future archeologists.